tox.ini

```diff
@@ -5,18 +5,9 @@
 
 [tox]
 envlist =
-          django1.11-py35,
-          django1.11-py36,
-          django1.11-py37,
-          django2.0-py35,
-          django2.0-py36,
-          django2.0-py37,
-          django2.1-py35,
-          django2.1-py36,
-          django2.1-py37,
-          django2.2-py36,
-          django2.2-py37,
-
+        django{1.11,2.0,2.1}-py{35,36,37},
+        django2.2-py{36,37},
```

Comment:
> Django 2.2 supports Python 3.5, 3.6, 3.7, 3.8 (as of 2.2.8), and 3.9 (as of 2.2.17). We highly recommend and only officially support the latest release of each series.
(c) https://docs.djangoproject.com/en/3.2/releases/2.2/

Reply:
I thus understand that I should change

`django2.2-py{36,37},`

for:

`django2.2-py{36,37,38,39},`

?

Thank you!

Reply:
Yes, please.
And thanks )

Reply:
This should be addressed in [580c27a](https://github.com/dominno/django-moderation/pull/200/commits/580c27a3f1e621adddbf8fec6146ae6e49b9ba6b). Best,
Martí